isotope/service/Dockerfile

```diff
@@ -9,9 +9,7 @@ COPY . /build/
 
 WORKDIR /build
 
-RUN  --mount=type=cache,target=/go/pkg/mod \
```

Comment:
can you keep the cache?

Reply:
Sure, it hasn't been working for me locally hence I removed it; will re-introduce it!

Reply:
Sure, will bring it in again; it wasn't working locally for me hence I removed it